### PR TITLE
chore: CI implementation

### PR DESCRIPTION
**Pull request by elivillalejos**
Hi! Patching from [this PR ](https://github.com/looker-open-source/viz-histogram-marketplace/pull/18)so I can fix the ci-workflow from Github as logs are no longer available in that one.

### CODE REVIEW

File: .github/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci-build-job:
+    uses: looker-open-source/reusable-actions/.github/workflows/marketplace-viz-ci-build.yml@main
```
Comment (yesudeep) [inline .github/ci-build.yml]: Let's ensure there is a blank line at the EOF throughout the PR.
Not having that prevents certain tools from working currectly on unix

**Review by yesudeep — CHANGES_REQUESTED** [review]

File: .github/ci-build.yml (hunk 1 above)
Comment (elivillalejos) [reply .github/ci-build.yml]: Addressed this in the last commit, thank you.

File: webpack.config.js [hunk 2]
```diff
@@ -40,4 +30,4 @@ var webpackConfig = {
   },
 };
 
-module.exports = webpackConfig;
+module.exports = webpackConfig;
```
Comment (yesudeep) [inline webpack.config.js]: nit: please ensure the file ends with a EOL character and doesn't have additional trailing newlines.

File: .github/ci-build.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (yesudeep) [inline .github/ci-build.yml]: nit: please ensure the file ends with a EOL character and doesn't have additional trailing newlines.

**Review by yesudeep — CHANGES_REQUESTED** [review]

**Review by yesudeep — APPROVED** [review]

**Pull request merged**